src/Database.js

```diff
@@ -167,8 +167,10 @@ class Database {
             reject(new Error(message));
           } else {
             /* eslint-disable no-param-reassign */
-            migration.up = up.replace(/^--.*?$/gm, '').trim();     // Remove comments
-            migration.down = down.replace(/^--.*?$/gm, '').trim(); // and trim whitespaces
+            migration.up = up.replace(/^-- Up.*?$/gm, '').trim();     // Remove comments
+            migration.up = migration.up.replace(/^-+$/gm, '').trim();
+            migration.down = down.replace(/^-- Down.*?$/gm, '').trim(); // and trim whitespaces
+            migration.down = migration.down.replace(/^-+$/gm, '').trim(); // and trim whitespaces
```

Comment:
Couldn't we just do `migration.down = down.trim();`? :thinking: 

This would prevent future issues with dashes in values.